### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_Alljobs_Screenreader:Voice Over read Table as a empty table.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5817]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS:MacOS Mojave 10.14.5
**Additional Details:**
Tool Name: Voice Over
**MAS Reference: MAS1.3.1**
**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Naviagte to the connected server->Select the database->Navigate to the SQL agent tab.
4.Navigate to jobs tab and a click on any table to navigate all jobs table
5. Observed that table is showing empty.
**Actual:**
issue1.Table is showing empty.
issue2.After applying shortcuts advised by voice over we are not able to navigate inside the table.

![image](https://user-images.githubusercontent.com/39204909/58774754-b48fc400-85e0-11e9-9695-c2694fb42145.png)

**Expected:**
Table should not show empty as we can see the table there and short cut should be applicable which is advised by the voiceover.
**User Impact:**
User with visual and motor disabilities will not be able to navigate through table.
**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx
**MAS Reference:**
MAS1.3.1
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference:**
[5817_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3245769/5817_AccessibilityA11y_AzureDataStudio_AllJobs_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
